src/main/java/DueDate.java

```diff
@@ -0,0 +1,6 @@
+/**
+ * DueDate is an interface for inheritance by tasks with a due date
+ */
+public interface DueDate {
```

Comment:
Good job on naming the interface correctly in PascalCase. I like your use of an interface here.